Bricks/strings/util.h

```diff
@@ -138,6 +147,23 @@ inline const T_OUTPUT& FromString(T_INPUT&& input, T_OUTPUT& output) {
                                                                                   output);
 }
 
+// Special case for `std::pair<>`. For Storage REST only for now.
+// TODO(dkorolev) + TODO(mzhurovich): Unify `FromString()` and `JSON()` under one `Serialize()` w/ policies?
+template <typename T_INPUT, typename T_OUTPUT_FST, typename T_OUTPUT_SND>
+struct FromStringEnumImpl<T_INPUT, std::pair<T_OUTPUT_FST, T_OUTPUT_SND>, false> {
+  static const std::pair<T_OUTPUT_FST, T_OUTPUT_SND>& Go(const std::string& input,
+                                                         std::pair<T_OUTPUT_FST, T_OUTPUT_SND>& output) {
+    const size_t dash = input.find('-');
```

Comment:
I don't get it: `ToStringImpl` combines with `:` and `FromStringEnumImpl` searches for `-` as a delimiter.